src/isomorphic/classic/__tests__/ReactContextValidator-test.js

```diff
@@ -261,7 +261,7 @@ describe('ReactContextValidator', () => {
     ReactTestUtils.renderIntoDocument(<Component testContext={{bar: 123}} />);
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(normalizeCodeLocInfo(console.error.calls.argsFor(0)[0])).toBe(
-      'Warning: Failed childContext type: ' +
+      'Warning: Failed child context type: ' +
```

Comment:
If this is released in 15.x, it would potentially break CIs on libraries that assert about this warning in tests.

We have some tests like this about prop types in React Redux so I wouldn't be surprised if somebody tested a similar thing about context types.